packages/engine-test-utils/src/__tests__/common-all/template.spec.ts

```diff
@@ -129,6 +129,41 @@ describe(`WHEN running applyTemplate tests`, () => {
       });
     });
 
+    describe("AND WHEN using capture helper", () => {
```

Comment:
very nitpick: `AND` usually combos with the THEN (the verification clause, as in doing multiple verifications) as opposed to the WHEN.